home/forms.py

```diff
@@ -19,6 +19,14 @@ class Meta:
             "content",
         )
 
+    def clean_sub_subject(self):
+        subject = self.cleaned_data.get("subject")
+        sub_subject = self.cleaned_data.get("sub_subject")
+        if sub_subject and sub_subject not in subject.sub_subject_set.all():
```

Comment:
I'm not sure I understood the check here.
I don't see the `sub_subject_set` field in the `Subject` model, How do you check that a specific sub-subject is related to a certain subject?

Reply:
it's because sub_sucject has subject field as ForeignKey so `sub_subject_set ` is automatic

Reply:
Just to make sure I understand, `sbj.sub_subject_set.all()` returns a list of all of the `sub_subject`s that has `sbj` as foreign key, right?

Reply:
yes